CaseDocs/atm_in_0001

```diff
@@ -32,50 +33,53 @@
  depvel_file		= '/glade/p/cesmdata/cseg/inputdata/atm/cam/chem/trop_mozart/dvel/depvel_monthly.nc'
  depvel_lnd_file		= '/glade/p/cesmdata/cseg/inputdata/atm/cam/chem/trop_mozart/dvel/regrid_vegetation.nc'
  exo_coldens_file		= '/glade/p/cesmdata/cseg/inputdata/atm/cam/chem/trop_mozart/phot/exo_coldens.nc'
- ext_frc_specifier		= 'H2O -> /glade/p/cesmdata/cseg/inputdata/atm/cam/chem/emis/elev/H2O_emission_CH4_oxidationx2_elev_1850-2100_CCMI_RCP8_5_c160219.nc',
-         'num_a1 -> /gpfs/fs1/p/acom/acom-climate/cmip6inputs/emissions_ssp119/emissions-cmip6-ScenarioMIP_IAMC-IMAGE-ssp119-1-1_num_so4_a1_anthro-ene_vertical_mol_175001-210012_0.9x1.25_c20181024.nc',
-         'so4_a1 -> /gpfs/fs1/p/acom/acom-climate/cmip6inputs/emissions_ssp119/emissions-cmip6-ScenarioMIP_IAMC-IMAGE-ssp119-1-1_so4_a1_anthro-ene_vertical_mol_175001-210012_0.9x1.25_c20181024.nc'
+ ext_frc_specifier		= 'H2O -> /glade/p/cesmdata/cseg/inputdata/atm/cam/chem/emis/elev/H2OemissionCH4oxidationx2_3D_L70_1849-2015_CMIP6ensAvg_c180927.nc',
```

Comment:
this is just a comment.  these namelists are pretty horrific and seem quite easily error prone in making a typo in the directory name or the filename.  is there any way to generate them with a directory name as a fixed string and sed'ing them to expand the names so it's easier to read?  not before we get the reanalysis running again - but something to look at later?  (in the interests of trying to make it easier to see errors)

Reply:
Tim and I know of no way to do that IN a namelist.
I do use that technique when I'm (re)creating the namelists
in the setup_* or other scripts.